android/src/main/java/com/appfolio/extensions/ContextExtensions.kt

```diff
@@ -61,3 +67,12 @@ fun Context.getUploadTask(
         null
     }
 }
+
+@SuppressLint("UnspecifiedRegisterReceiverFlag")
+fun Context.registerReceiverCompat(receiver: BroadcastReceiver, filter: IntentFilter) {
+    if (SDK_INT >= 34) {
```

Comment:
Do you mind linking any doc on why we need to make a different call based on SDK version?